packages/base/src/commands.ts

```diff
@@ -205,18 +224,21 @@ export async function executeOperator(
     ...currentJcadContent,
     objects: [...currentJcadContent.objects, objectModel]
   };
-  const dryRunResult =
-    await current.content.currentViewModel.dryRun(updatedContent);
-  if (dryRunResult.status === 'error') {
-    showErrorMessage(
-      `Failed to create the ${name} operation`,
-      `The ${name} tool was unable to create the desired shape due to invalid parameter values. The values you entered may not be compatible with the dimensions of your piece.`
-    );
 
+  const dryRunResult = await dryRunCheck({
+    jcadContent: updatedContent,
+    mainView: current.content.currentViewModel,
+    requestedOperator: name
+  });
+  if (!dryRunResult) {
     return;
```

Comment:
Would it be nice to also throw an error here in case someone want to debug this in future?

Reply:
The `dryRunCheck` function has already raised the error.

Reply:
Oh yes. Oops
That descriptive error will be thrown and will return null. That makes sense. Thanks 😄